openassessment/xblock/static/sass/oa/elements/_staff.scss

```diff
@@ -44,6 +61,13 @@
     color: $copy-staff-color;
   }
 
+  .ui-staff__subtitle {
+    @extend %t-subheading;
+    @include fontSize($f-size-medium);
+    font-weight: 600;
```

Comment:
Actually, I wasn't reading the inspector correctly. It looks like it is in fact working.